recipes/libmaxminddb/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+import os
+import shutil
+from conans import ConanFile, CMake, tools
+
+
+class libmaxminddbConan(ConanFile):
+    name = "libmaxminddb"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://maxmind.github.io/libmaxminddb/"
+    description = "C library for the MaxMind DB file format"
+    topics = ("MaxMind", "GeoIP")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    exports_sources = "patches/**"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("libmaxminddb-{}".format(self.version), self._source_subfolder)
+
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        cmake = CMake(self)
+        cmake.definitions["BUILD_TESTING"] = "OFF"
+        cmake.configure(source_folder=self._source_subfolder, build_folder=self._build_subfolder)
```

Comment:
```suggestion
        cmake.configure(build_folder=self._build_subfolder)
```

You have to provide CMake wrapper. Please, read https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md#how-to-provide-a-good-recipe

Reply:
Thank you for the pointers.

The library doesn't work with the CMake wrapper out of the box. The build seems to set `INTERFACE_INCLUDE_DIRECTORIES` to include the source directory incorrectly and therefore the configuration fails. I will have a look at that later.

Reply:
This was indeed an upstream bug. It's fixed now with https://github.com/maxmind/libmaxminddb/pull/259